stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
I did the research and came to my conclusions. Yes, there is a limited spectrum of colors: the default (yellow) and this one. That's the reason I didn't add any other modifier.

My point is that there is no advantage in having a semantic name, like the usual `alternative`, over the color's name.